src/state/process.rs

```diff
@@ -26,6 +26,8 @@ use crate::{Codec, Consensus, ConsensusResult, Crypto, INIT_EPOCH_ID, INIT_ROUND
 
 const CHECK_EPOCH_SUCCESS: bool = true;
 const CHECK_EPOCH_FAILED: bool = false;
+const FUTURE_EPOCH_GAP: u64 = 5;
```

Comment:
Is this value defined in the Overlord protocol?

Reply:
yes